source/sonata_simulation.rst

```diff
@@ -267,6 +267,7 @@ Spike events are created from the cells indicated in a file and delivered to the
    ============================== ========== ============ ==========================================
    spike_file                     text       Mandatory    Indicates the location of the file with the spike info for injection.
    source                         text       Optional     The node set to replay spikes from.
+   connection_type                text       Optional     Type of the connectivity between the source and target of injection. Supported values are :ref:`the connection types of populations<edge_pop_type>`: "chemical", "electrical", "synapse_astrocyte", "endfoot", "neuromodulatory", "point_neuron". Default is "chemical".
```

Comment:
In BlueConfig we have a key (named `Type`) for the `synapse_replay` stimulus to specify the connectivity type to receive the injection. It should be migrated to SONATA. The values could be the same as the connection types of the population as already defined (in `sonata_config.rst`). However, we also need the `point_neuron` (or some other naming) connectivity type which is not defined by `sonata_config.rst`.

Reply:
Hello @mgeplf , after talking to @iomaganaris , I've added a new edge population type `point_neuron` in sonata circuit config file for point neuron connectivity.  In sonata sim config, we will use a new key `connection_type` in `inputs/synapse_replay` to point to the edge population type which should receive the replay.

Please let us know what you think about that new type.

Reply:
I think there already was a type, I see `TM_synapse` here:
https://sonata-extension.readthedocs.io/en/latest/sonata_tech.html#fields-for-point-neuron-connectivity

Or are these different?  I personally don't find the `TM_synapse` name very intuitive, but I guess there are different kinds of “point neuron connectivity”?

Perhaps @jdcourcol also has an idea?

Reply:
Hello @mgeplf @jdcourcol ,

From @jamesgking and @iomaganaris,  I've learnd that `TM_synapse` is one type of point neuron synapses used by the AdEx model. There are other types, such as the placeholder models. Therefore, `point_neuron` is better to describe all types of point neuron connectivity.  We can rephrase https://sonata-extension.readthedocs.io/en/latest/sonata_tech.html#fields-for-point-neuron-connectivity so that "Connection type is `point_neuron`" and the current list of synapse fields are for `Tsodyks Markram model`. The list might be extended later if other models request new fields. What do you think?

Reply:
Is the list different for different point neuron models?  If so, I think it would makes sense to have a separate type for each; maybe something like: `point_neuron_tsodyks_markram`, and the another one `point_neuron_integrate_and_fire`?

Reply:
I would rename it "point_neuron_tsodyks_markram" as suggested by @mgeplf .
We can introduce new connection type (and therefore different fields) if we cross that bridge.

Reply:
`point_neuron_tsodyks_markram` sounds good as the connection type in edges.
Then for the `model_type` in node files, is it better to be specific too? How about `point_neuron_adex` instead of `point_neuron` ?
https://sonata-extension.readthedocs.io/en/latest/sonata_tech.html#fields-for-point-neuron-population-model-type-point-neuron

Reply:
I agree, this makes sense to me.

Reply:
Great, I've just committed a change to use `point_neuron_adex` as model_type and `point_neuron_tsodyks_markram` as connection type in `sonata_config.rst` and `sonata_tech.rst`. Please have a look. Thanks!